DataFormats/TrackerRecHit2D/src/classes_def.xml

```diff
@@ -12,7 +12,7 @@
    <version ClassVersion="10" checksum="877294307"/>
   </class>
   <class name="SiTrackerGSMatchedRecHit2D" ClassVersion="14">
-    <version ClassVersion="14" checksum="3882739671"/>
+    <version ClassVersion="14" checksum="3518158739"/>
```

Comment:
Is there a problem here? Shouldn't the version number and checksum change together?